tests/testAll.py

```diff
@@ -21,6 +21,7 @@
     "testEupspkg",
     "testSetups",
     "testEupsIntegration",
+    "testSpaces",
```

Comment:
It would be better to add a `testSpaces.sh` script into the existing `testEupsIntegration` suite.